papaya-web/src/main/java/pl/edu/agh/papaya/notification/message/Message.java

```diff
@@ -0,0 +1,41 @@
+package pl.edu.agh.papaya.notification.message;
+
+import java.time.LocalDateTime;
+import pl.edu.agh.papaya.model.NotificationType;
+
+public interface Message {
```

Comment:
Treść tego powiadomienia niekoniecznie jest SUPER - i jest po angielsku, a widoczki mamy po polsku, ale nie wiem jak to ładnie zrobić  - pewnie wyjąć do plików czy coś, ale trochę nie umiem

Reply:
aplikacja też jest częściowo po angielsku ;)